test/test_torch.py

```diff
@@ -1769,6 +1770,23 @@ def matrixmultiply(mat1, mat2):
             _test_mm(n, m, p, torch.int32, lambda x, y: torch.randint(0, 100, (x, y), dtype=torch.int32))
             _test_mm(n, m, p, torch.int64, lambda x, y: torch.randint(0, 100, (x, y), dtype=torch.int64))
 
+    @unittest.skipIf(
+        not TEST_NUMPY or not TEST_SCIPY, "Numpy or Scipy not found")
+    def test_gelu(self):
+        def _test_gelu(n, m, dtype):
+            def _gelu_ref(X):
+                return X * norm.cdf(X)
+
+            X = np.random.randn(n, m).astype(dtype)
```

Comment:
Can you also test non-contiguous tensors? Like
```
x = torch.rand(50, 50)[:, ::2]
```

Reply:
It looks like this always coverts the input to be contiguous, although it probably doesn't need to since it's now using TensorIterator.

Reply:
I added test for non-contiguous inputs.